### ISSUE TITLE
Can't Download Latest Podcast or Stream File

### ISSUE THREAD

**Issue by mowestusa** [thechangelog/changelog.com#346]
I was notified of a new episode of Changelog Podcast in AntennaPod, my Podcast client that I use Android. However, when I go to download the latest episode "What the web could be..." or any episode the download fails and streaming the podcast in AntennaPod also fails. Perhaps something happened with your storage or podcast links recently that are in the RSS feed.

**Comment by jerodsanto** [comment]
Hmm, we've had no other reports and I know the episode (and others in the feed) is available on other podcast apps.

Maybe reach out to the [AntennaPod folks](https://github.com/AntennaPod/AntennaPod) and see if they can help diagnose the problem?

**Comment by mowestusa** [comment]
Jerod, I just switched over to my work network, and was able to download the episode. This leads me to believe the problem was with my PiHole server that I have set up on my home network which is not on my work network. Is it possible that something changed that recently that could have gotten your feed on a PiHole block list?

This was the full error I got from AntennaPod:
```
Failed to connect to chtbl.com/0.0.0.0:443 

File URL:
https://chtbl.com/track/382DG4/https://cdn.changelog.com/uploads/podcast/426/the-changelog-426.mp3
```

Since this appears to be an issue with my PiHole, I will try whitelisting your URL and try again on my home network, and I will mark this closed since it doesn't appear to be an issue on your end. Thanks for responding so quickly.

**Comment by jerodsanto** [comment]
Interesting. We're using [Chartable](https://chartable.com) at the moment as a sanity check on our download/reach stats (which is home brewed by me), so that's why our feed for _The Changelog_ redirects through `chtbl.com`.

It's possible that they got on PiHole's blacklist somehow. We don't use them for any of our other podcast feeds, so try downloading an episode of [Go Time](https://gotime.fm) or [JS Party](https://jsparty.fm/) and see if it works. That would be somewhat of a smoking gun, if so.

**Comment by mowestusa** [comment]
Well, I was going to do some testing tonight to see if Chartable was blacklisted on the PiHole, but when I downloaded an episode of Changelog tonight everything worked just fine. When I looked at the logs in PiHole it said that your chartable URL was whitelisted so I'm not sure what happened. I didn't update anything on the PiHole so its setting could not have changed. The feed just didn't want to work with my home network that day for that period of time. Everything seems to be working fine again.

**Comment by jerodsanto** [comment]
Well, not the most satisfying conclusion to this little mystery, but happy to hear everything is fine now. 💚

**Comment by adamstac** [comment]
Yup, just to clarify these are the two default ad lists that Pi-hole uses:

- `https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts`
- `https://mirror1.malwaredomains.com/files/justdomains` (this seems to be 404'ing so I have this one disabled)

The good news is that `chtbl.com` isn't in Pi-hole's ad lists.

**Comment by mowestusa** [comment]
@adamstac as I mentioned in the comment above, `chtbl.com` was not blacklisted on my Pi-Hole server. I thought my Pi-Hole might have been blocking the domain because I could download Changelog on my network at work which doesn't use Pi-Hole but not at home. I have added a number of additional block lists to my Pi-Hole, but I did verify that non of the block lists that I installed are blocking `chtbl.com` and I can suddenly download Changelog again at home without making any modifications to my network or Pi-Hole. Sometimes technology is just weird. Thanks for all of the advice and help in looking for the cause, it is always weird when things seem to fix themselves.
